File: ex1.py
def starPalindrome(n):
    for i in range(1,n+1):
        print(" "*(n-i),"*"*(i*2-1),end="\n")

    for j in range(n-1,0,-1):
        print(" "*(n-j),"*"*(j*2-1),end="\n")

rows=int(input("enter number of rows: "))
starPalindrome(rows)

File: ex1.js
function starPalindrome(n) {
    for (let i = 1; i <= n; i++) {
        console.log(" ".repeat(n - i) + "*".repeat(i * 2 - 1));
    }

    for (let j = n - 1; j > 0; j--) {
        console.log(" ".repeat(n - j) + "*".repeat(j * 2 - 1));
    }
}

rows = parseInt(prompt("Enter number of rows: "));
starPalindrome(rows);
